lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
@jannyHou Also, let me confirm that I don't get this error on doing http request:
`Uncaught TypeError: Cannot read property 'posts' of undefined
at MongoDB.all (lib/mongodb.js:856:62)`
and if I do a:
`var relations = this._models[model].settings.relations;`
then a: 
`console.log(relations)`
I get the relations object successfully, that's only if it's called via an http request